.github/workflows/examine-java-bindings.yml

```diff
@@ -78,7 +78,8 @@ jobs:
         working-directory: client/bindings/java/
         run: |
           chmod +x gradlew
-          ./gradlew test -i
+          # Only run test that doesn't require a node 
+          ./gradlew test -i --tests "*testAddressDerivation"
```

Comment:
But we're just ignoring tests now? That doesn't sound like solving the actual issue :(

Reply:
The main issue was that we sent too many requests to the faucet (the rate limit got changed, probably that's why we get rate limited more easily now)
Already reduced the amount of requests in requestTestFundsFromFaucet, but in the logs you can still see
```
    null
    {"error":{"code":"500","message":"internal server error. error: %!s(\u003cnil\u003e)"}}
```
for the faucet requests which is returned when we're rate limited, of course I could increase maxAttemts to 1000 or something so it will work at some point, but not sure if that's really better then

Also the Rust tests should already cover almost everything, so I think we don't need to test the Java tests in the CI every time

Reply:
And we're 100% sure we're not spamming somehow? If the rust tests work, the Java test should work as well...

Reply:
At the very least I don't think we should do `--tests "*testAddressDerivation"`, it seems very error prone if we add more tests. Can't we just ignore some tests?

Reply:
I think we can add `@ignore`, but we would need to add this to basically all tests and I haven't seen a way to run them then locally

Reply:
But if we can't even run it (I'm not 100% sure, but haven't seen any option to do so), then we could also completely remove them since they would never be executed unless you comment this

Reply:
Changed it so we run all tests with the private tangle on ubuntu and only the address test on macos and windows because the private tangle setup doesn't easily work there
This way all things should be tested on ubuntu, but we still know that it also builds on macos and windows, so I think that's okay